src/components/Preferences/ScheduleSelector.tsx

```diff
@@ -30,22 +30,25 @@ const Timetable = (props) => {
     };
 
     function handleChange(newSchedule) {
-        const json_arr = [];
-        newSchedule.forEach((element) => {
-            const time = {
-                day: element.getDay(),
-                time: element.getHours(),
-            };
-            json_arr.push(time);
```

Comment:
Sorry I think there's merge conflicts because I changed these variable names to camel case.

Reply:
yeah there will be conflicts in these files
<img width="348" alt="image" src="https://user-images.githubusercontent.com/38937780/175834039-6287101a-9a55-40de-bd29-1304a4f43da2.png">